frontend/src/components/DeletedNote.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from "../styles/DeletedNote.module.css";
+import stylesUtils from "../styles/utils.module.css";
+import { Card } from "react-bootstrap";
+import { DeletedNote as DeletedNoteModel } from "../models/deletedNote";
+import { MdDelete, MdOutlineSettingsBackupRestore } from "react-icons/md";
+
+function formatExpiresIn(deletedAt: string): string {
+  const now = new Date();
+  const deletedAtDate = new Date(deletedAt);
+
+  const expiresAt = new Date(deletedAtDate.getTime() + 24 * 60 * 60 * 1000);
+  const timeDifference = expiresAt.getTime() - now.getTime();
+
+  if (timeDifference <= 0) {
+    return "Expired";
+  }
+
+  const minutesDifference = Math.floor(timeDifference / (1000 * 60));
+  const hoursDifference = Math.floor(minutesDifference / 60);
+
+  if (hoursDifference >= 1) {
+    return `${hoursDifference}h`;
+  } else {
+    return `${minutesDifference}m`;
+  }
+}
+
+interface DeletedNoteProps {
+  note: DeletedNoteModel;
+  onDeleteNoteClicked: (note: DeletedNoteModel) => void;
+  onRestoreNoteClicked: (note: DeletedNoteModel) => void;
+  className?: string;
+}
+
+const DeletedNote = ({
+  note,
+  onDeleteNoteClicked,
+  onRestoreNoteClicked,
+  className,
+}: DeletedNoteProps) => {
+  const { title, text, deletedAt } = note;
+
+  const ExpiresIn = "Expires in: " + formatExpiresIn(deletedAt);
+
+  return (
+    <Card className={`${styles.noteCard} ${className}`}>
+      <Card.Body className={styles.cardBody}>
+        <Card.Title className={stylesUtils.flexCenter}>
+          {title}
+          <div className="text muted ms-auto">
+            <MdOutlineSettingsBackupRestore
+              onClick={(e) => {
+                onRestoreNoteClicked(note);
```

Comment:
You don't have to add `...Clicked` for the onClick event handlers, you can simply name it `onRestore`

Reply:
bc we all know that it's a note